spec/dummy/app/avo/resources/product_resource.rb

```diff
@@ -11,13 +11,22 @@ class ProductResource < Avo::BaseResource
       },
       content: {
         classes: "bg-gray-50 !text-pink-600"
+      },
+      wrapper: {
+        classes: "bg-gray-50 !text-pink-600"
       }
     }
   }
   field :description, as: :trix
   field :image, as: :file, is_image: true
   field :price, as: :number
   field :category, as: :select, enum: ::Product.categories
+  field :vat_tax, as: :select, options: {
```

Comment:
- rm migration and field and make select options with temporary fields in city population.

- test filters with enum array and with enum hash